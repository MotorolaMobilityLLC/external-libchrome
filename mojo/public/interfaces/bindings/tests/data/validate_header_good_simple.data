[dist4]message_header  // num_bytes
[u4]2                  // num_fields
[u4]0                  // name
[u4]0                  // flags
[anchr]message_header
