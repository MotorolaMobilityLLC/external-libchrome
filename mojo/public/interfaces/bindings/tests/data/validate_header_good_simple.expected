num_bytes: 16
num_fields: 2
name: 0
flags: 0
